svthalia/concrexit
# Add registration conformation email

Closes #2310



### Summary
Added option to get registrations conformations via email in the profile  and if it is turned on 
users will receive a conformation of registration via email.

### How to test
Steps to test the changes you made:
1.  Make sure the check mark to receive conformation emails is on in profile
2.  Go to Calendar on the site
3.  Click on Register
4.  Turn the check-mark off
5. register again for an event
